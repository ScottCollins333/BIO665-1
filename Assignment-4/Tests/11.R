source("11.R")
