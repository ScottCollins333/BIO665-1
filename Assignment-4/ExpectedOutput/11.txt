[1] 1.844674e+19
